src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/HostBuilderTests.cs

```diff
@@ -572,6 +572,25 @@ public void HostBuilderConfigureDefaultsInterleavesMissingConfigValues()
             Assert.Equal(expectedContentRootPath, env.ContentRootPath);
         }
 
+        [Fact]
+        public void HostBuilderCanConfigureBackgroundServiceExceptionBehavior()
+        {
+            using IHost host = new HostBuilder()
+                .ConfigureServices(
+                    services =>
+                        services.Configure<HostOptions>(
+                            options =>
+                            options.BackgroundServiceExceptionBehavior =
+                                BackgroundServiceExceptionBehavior.StopHost))
```

Comment:
I have tests for both `Ignore` and `StopHost` already, both explicitly set their intentions, and test each expected case. I think this one is fine as is. What if the default changes, I'd rather have a test case in place that explicitly validates this condition - do you agree?

Reply:
> Maybe we should switch this to `Ignore` since `StopHost` is the default.

I'm updating this to be a `Theory` instead, going to test both cases explicitly.